reactive/kubernetes_master.py

```diff
@@ -126,11 +127,14 @@
 kubecontrollermanagerconfig_path = "/root/cdk/kubecontrollermanagerconfig"
 kubeschedulerconfig_path = "/root/cdk/kubeschedulerconfig"
 cdk_addons_kubectl_config_path = "/root/cdk/cdk_addons_kubectl_config"
+cdk_logs = "/var/log/cdk/"
```

Comment:
Similar comment as https://github.com/charmed-kubernetes/kubernetes-docs/pull/617#pullrequestreview-795425141:

I wonder since we're making this change if we should drop cdk in favor of ck. ck is awfully short and could confuse people wondering what's this ck subdirectory for? so perhaps we should consider charmed-kubernetes as our subdir name and ck in filenames?